### DISCUSSION TITLE
Different color marking for Boolean column value(s)

### DISCUSSION THREAD

**Discussion by kseniiaguzeeva** [dbeaver/dbeaver#10644] [Ideas]
All view-mode customisations should be configurable via wide "Settings..." menu
Someone like strict, other - decorated (even on their own design-way(!)).

**Comment by kseniiaguzeeva** [comment]
by maciey:
"It is better but still Id prefer previous (bolded version)

Ive tried setup different colours but still not ideal

![image](https://user-images.githubusercontent.com/51405061/101621547-f1831500-3a26-11eb-897a-e327ef41a088.png)

obraz
![image](https://user-images.githubusercontent.com/51405061/101621574-fcd64080-3a26-11eb-90f9-eabc5cec4036.png)


in previous versions:
obraz"
![image](https://user-images.githubusercontent.com/51405061/101621598-052e7b80-3a27-11eb-8787-0b64633cfc11.png)

**Comment by serge-rider** [comment]
From #9327
